echopype/convert/parse_azfp.py

```diff
@@ -156,48 +254,36 @@ def compute_battery(N):
                             self._print_status()
                         # Compute temperature from unpacked_data[ii]['ancillary][4]
                         self.unpacked_data["temperature"].append(
-                            compute_temp(self.unpacked_data["ancillary"][ping_num][4])
+                            self._compute_temperature(ping_num, temperature_is_valid)
                         )
                         # compute x tilt from unpacked_data[ii]['ancillary][0]
                         self.unpacked_data["tilt_x"].append(
-                            compute_tilt(
-                                self.unpacked_data["ancillary"][ping_num][0],
-                                self.parameters["X_a"],
-                                self.parameters["X_b"],
-                                self.parameters["X_c"],
-                                self.parameters["X_d"],
-                            )
+                            self._compute_tilt(ping_num, "Y", tilt_x_is_valid)
```

Comment:
```suggestion
                            self._compute_tilt(ping_num, "X", tilt_x_is_valid)
```

Reply:
many bugs... I have done this too quickly!! and too bad that tests do not catch it, because the tests only checks the backscatter data.